.github/CONTRIBUTING.md

```diff
@@ -193,3 +193,5 @@ We've written some convenience helper modules and functions to help with API tes
 - `CodeCorpsWeb.ApiCase` - used to simplify testing JSON API endpoints
 - `CodeCorps.AuthenticationTestHelpers` - provides authentication helpers for authenticating a `conn`
 - `CodeCorps.Factories` - provides factories using [`ex_machina`](https://github.com/thoughtbot/ex_machina), which makes it easy to create test data and associations with Ecto
+- `CodeCorps.GitHub.TestHelpers` - provides test helpers for loading GitHub API endpoint or event JSON fixtures, as well as mocking the API
+- `CodeCorps.GitHub.SuccessAPI` - provides some automatic mocking of the GitHub API, or raises an exception if you have an unhandled request to the API
```

Comment:
Although this is documented here, I am concerned that this is not more explicit right now, as it tripped me up how I'm supposed to do the API mocking and took some digging.

Reply:
Other than setting up the testing framework in each test module, I can't think of a way to make it more explicit.

We could add more info to the error message, maybe?